Files/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -154,8 +154,8 @@
             </MenuFlyoutItem>
         </MenuFlyout>
 
-        <MenuFlyout x:Key="BaseLayoutItemContextFlyout" Opening="RightClickItemContextMenu_Opening">
-            <MenuFlyoutItem
+            <MenuFlyout x:Key="BaseLayoutItemContextFlyout" x:Name="BaseLayoutItemContextFlyout" Opening="RightClickContextMenu_Opening">
```

Comment:
Is this correct? Shouldn't this be:
```cs
<MenuFlyout 
    x:Key="BaseLayoutItemContextFlyout" 
    x:Name="BaseLayoutItemContextFlyout" 
    ----> Opening="RightClick***Item***ContextMenu_Opening">
```

Reply:
No, that one is correct.

Reply:
My apologies you are right.  https://github.com/files-community/files-uwp/issues/1185 will address this.